React/React hooks.md

```diff
@@ -1,7 +1,68 @@
 ### React hooks
 
-#### Using hook dependency arrays
-Dependency arrays are a way to trigger re-firing of hooks only when it's dependencies change. React will keep the values generated by a hook stored in memory and then run it when a change of a dependency has occured. This applies to all hooks that can have dependencies.
+React Hooks were introduced in version 16.8.0 as Function Component counterpart of Class Component Lifecycle.  
+For more information, see the official [React Hooks API Reference](https://reactjs.org/docs/hooks-reference.html).
+
+#### Hooks flow
+Before we start using hooks in React we need to understand the flow.
```

Comment:
```suggestion
Before starting, it is important to understand the flow of react hooks.
```

These suggestions are also "meh", but we should  somehow avoid repeating ourselves too much in consecutive sentences.